modules/ossm-rn-new-features.adoc

```diff
@@ -15,6 +15,164 @@ Module included in the following assemblies:
+
+* About migrating to a cluster-wide mesh
+* Excluding namespaces from a cluster-wide mesh
+* Defining which namespaces receive sidecar injection in a cluster-wide mesh
+* Excluding individual pods from a cluster-wide mesh
+
+=== {SMProductName} operator on ARM-based clusters
+This release provides the {SMProductName} Operator available on ARM-based clusters as a generally available feature. Images are available for Istio, Envoy, Prometheus, Kiali, Jaeger, and Grafana. 
+
+=== OpenShift Service Mesh Console plugin
+
+This release introduces a generally available version of the OpenShift {SMProductShortName} Console (OSSMC) plugin. 
+
+The OSSMC plugin is an extension to the OpenShift Console that provides visibility into your Service Mesh. With the OSSMC plugin installed, a new Service Mesh menu option appears on the left-hand navigation of the web console, as well as new Service Mesh tabs that enhance existing Workloads and Service console pages. 
+
+The features of the OSSMC plugin are very similar to those of the standalone Kiali Console. The OSSMC plugin does not replace the Kiali Console, and after installing the OSSMC plugin, you can still access the standalone Kiali Console. 
+
+=== Istio OpenShift Routing (IOR) default setting change
+The default setting for Istio OpenShift Routing (IOR) has changed. The setting is now disabled by default.
+
+You can use IOR by setting the `enabled` field to `true` in the `gateways.openshiftRoute` specification of the `ServiceMeshControlPlane` resource.
+
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+spec:
+  gateways:
+    openshiftRoute:
+      enabled: true
+----
+
+=== Gateway API CRD versions
+:FeatureName: {product-title} Gateway API support
+include::snippets/technology-preview.adoc[]
+
+A new version of the Gateway API CRD is now available. Refer to the following table to determine which Gateway API version should be installed with the OpenShift {SMProductShortName} version you are using:
+
+|===
+|Service Mesh Version | Istio Version | Gateway API Version | Notes
+
+|2.5.x
+|1.18.x
+|0.6.2
+|Use experimental branch because `ReferenceGrand` is missing in v0.6.2
+
+|2.4.x
+|1.16.x
+|0.5.1
+|For multitenant mesh deployment, all Gateway API CRDs must be present. Use experimental branch.
+|===
+
+=== Integration with {DTProductName} (Tempo) Stack
+:FeatureName: {product-title} {DTProductName} (Tempo) stack
+include::snippets/technology-preview.adoc[]
+
+This release introduces a technology preview integration of the tracing extension provider(s). You can expose tracing data to the {DTProductName} (Tempo) stack by appending a named element and `zipkin` provider to the `spec.meshConfig.extensionProviders` specification. Then, a telemetry custom resource configures Istio proxies to collect trace spans and send them to the Tempo distributor service endpoint.
```

Comment:
There is a discussion about GA this integration in 2.5 release note. We may replace or remove the "technology preview" phrase this line.

Reply:
Done. It's now listed as GA.